apps/omg_watcher/lib/db/transaction.ex

```diff
@@ -142,7 +142,7 @@ defmodule OMG.Watcher.DB.Transaction do
         ]
       )
 
-    _ = Logger.info("Block ##{block_number} persisted in DB done in #{insert_duration / 1000}ms")
+    _ = Logger.debug("Block \##{block_number} persisted in WatcherDB, done in #{insert_duration / 1000}ms")
```

Comment:
Do we need to exit # character?
Why don't we pass a function as argument to Logger.debug?

Reply:
> Do we need to exit # character?

I think not, I'm doing it more to make it look intentional :). Then, I'm keeping it to be somewhat consistent + don't need to modify in too many places now

> Why don't we pass a function as argument to Logger.debug?
it is not necessary anymore for lazyness (since Elixir 1.7). So we went for the more succinct and less confusing syntax.